File: ios/PackageInstaller.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(PackageInstaller, NSObject)

RCT_EXTERN_METHOD(install:(NSString *)path withPackageName:(NSString *)packageName
                 withResolver:(RCTPromiseResolveBlock)resolve
                 withRejecter:(RCTPromiseRejectBlock)reject)

+ (BOOL)requiresMainQueueSetup
{
  return NO;
}

@end

File: ios/PackageInstaller.swift
@objc(PackageInstaller)
class PackageInstaller: NSObject {

    @objc(install:withPackageName:withResolver:withRejecter:)
    func install(path: String, packageName: String, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
    resolve(false)
  }
}
